src/site/layouts/step_by_step.drupal.liquid

```diff
@@ -0,0 +1,71 @@
+{% include "src/site/includes/header.html" with drupalTags = true %}
+
+{% include "src/site/includes/alerts.drupal.liquid" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+
+<div id="content" class="interior">
+  <main class="va-l-detail-page">
+    <div class="usa-grid usa-grid-full">
+      <div class="usa-width-three-fourths">
+        {% if !entityPublished %}
+          <div class="usa-alert usa-alert-info">
+            <div class="usa-alert-body">
+              <p class="usa-alert-text">You are viewing a draft.</p>
+            </div>
+          </div>
+        {% endif %}
+
+        <article class="usa-content">
+          <!-- Title -->
+          <h1>{{ title }}</h1>
+
+          <!-- Intro -->
+          <p>{{ fieldIntroTextLimitedHtml.processed }}</p>
+
+          <!-- Alert -->
+          {% if fieldAlertSingle %}
+            {% include "src/site/paragraphs/alert_single.drupal.liquid" with entity = fieldAlertSingle.entity %}
+          {% endif %}
+
+          <!-- Buttons -->
+          {% for fieldButton in fieldButtons  %}
+            {% include "src/site/paragraphs/button.drupal.liquid" with entity = fieldButton.entity %}
+          {% endfor %}
+
+          <!-- Steps -->
+          <h2>Step-by-step</h2>
+          <ol class="process">
+            {% assign count = 1 %}
+            {% for fieldStep in fieldSteps.entity.fieldStep %}
+              <li class="process-step list-{{ count | numToWord }}">
+                {{ fieldStep.entity.fieldWysiwyg.processed }}<br/>
+                <img alt="{{ fieldStep.entity.fieldMedia.entity.thumbnail.alt }}" src="{{ fieldStep.entity.fieldMedia.entity.thumbnail.url }}" />
```

Comment:
I think you can make use of the template `src/site/paragraphs/media.drupal.liquid` here instead of the img element

Reply:
oh cool!

Reply:
I replaced this line with `{% include "src/site/paragraphs/media.drupal.liquid" with entity = fieldStep.entity.fieldMedia.entity %}` and the image doesn't render anymore 🤔